quick_classifier.py

```diff
@@ -136,11 +164,14 @@ def bottom_k_scores(scores, k):
                 keyword_list = [kw['category'] for kw in keywords]
                 score_list = [kw['score'] for kw in keywords]
 
-
-            # header = 'bibcode,title,abstract,keywords,scores,threshold'
-            record_output = [record['bibcode'],record['title'],record['abstract'],', '.join(map(str, keyword_list)),', '.join(map(str, score_list)),str(threshold)]
-
-            output_list.append(record_output)
+                for keyword in keywords:
+                    print()
```

Comment:
Remove prints from final commit

Reply:
Print lines cleaned up.  Command line option (`-v` or `--verbose`) added to allow more detailed, record level information to be printed to screen.